index.d.ts

```diff
@@ -1119,12 +1124,17 @@ export interface Data<T extends FirestoreTypes.DocumentData> {
 }
 
 export namespace FirebaseReducer {
-  export interface Reducer<ProfileType = {}> {
```

Comment:
This is breaking change. The parameter order was changed.

Reply:
Good call out. Yeah, this was in an attempt to have the feature work as originally intended